src/vision/status_quo/frederik_makes_their_first_steps_into_async.md

```diff
@@ -0,0 +1,64 @@
+However, since they have heard that async is suitable for fast networked applications, they decide to try using async for their distributed database.
+After all, a fast networked application is exactly what they are trying to make.
+
+To further solidify the decision of using async, Frederik goes looking for some information and opinions on async in Rust. Doubts created by reading some tweets about how most people should be using threads instead of async for simplicity reasons are quickly washed away by helpful conversations on the Rust discord.
+
+### Learning about Async
+Still enarmored with the first edition of the Rust book, they decide to go looking for an updated version, hoping that it will teach them async in the same manner that it taught them so much about the language and design patterns for Rust. Dissapointed, they find no mention of async in the book, aside from a note that it exists as a keyword.
+
+Not to be deterred, they go looking further, and start looking for similarly great documentation about async.
+After stumbling upon the async book, their dissapointment is briefly replaced with relief as the async book does a good job at solidifying what they have already learned in various blog posts about async, why one would use it and even a bit about how it all works under the hood.
+They skim over the parts that seem a bit too in depth for now like pinning, as they're looking to quickly get their hands dirty.
+Chapter 8: The Async Ecosystem teaches them what they already picked up on through blog posts and contentious tweets: the choice of the runtime has large implications on what libraries they can use.
+
+### The wrong time for big decisions
+Frederik's dreams to quickly get their hands dirty with async Rust are shattered as they discover that they first need to make a big choice: what executor to use. Having had quite a bit of exposure to the conversations surrounding the incompatible ecosystems, Frederik is perhaps a bit more paranoid about making the wrong choice than the average newcomer.
+This feels like a big decision to them, as it would influence the libraries they could use and switching to a different ecosystem would be all but impossible after a while. Since they would like to choose what libraries they use before having to choose an executor, Frederik feels like the decision making is turned on its head. 
+
+Their paranoia about choosing the right ecosystem is eased after a few days of research, and some more conversations on the Rust subreddit, after which they discover that most of the RPC libraries they might want to use are situated within the most popular Tokio ecosystem anyways. Tokio also has a brief tutorial, which teaches them some basic concepts within Tokio and talks a bit more about async in general.
+
+### Woes of a newcomer to async
```

Comment:
It seems like there are two large questions posed by this story, and I'm wondering if they should be broken up into two stories:
* Getting started means almost immediately reckoning with the choice of runtime.
* How to mix blocking and non-blocking apis.

Reply:
I think its ok to merge as is, there could and should be more stories focusing on the blocking/non-blocking mix in particular, it doesn't feel like the primary focus here